### PR TITLE
Eelbrain

### PR DESCRIPTION
**Pull request by christianbrodbeck**

### CODE REVIEW

File: recipes/eelbrain/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - {{ compiler('c') }}
+    - setuptools >=17
+  host:
+    - python
+    - python.app  # [osx]
+    - olefile  # [win or linux]
+    - pywin32-ctypes  # [win]
+    - pip
+    - cython >=0.21
+    - numpy 1.11.*
```
Comment (chrisburr) [inline recipes/eelbrain/meta.yaml]: Is it necessary to force numpy 1.11 here?

```suggestion
    - numpy
```

Comment (christianbrodbeck) [reply recipes/eelbrain/meta.yaml]: Thanks @chrisburr, I was advised to build against the lowest compatible numpy version, hence `1.11`, but maybe that advice is outdated now? (see https://github.com/conda/conda-build/issues/2717#issuecomment-369316492)

Comment (chrisburr) [reply recipes/eelbrain/meta.yaml]: I think this is now handled by default, [see here](https://conda-forge.org/docs/meta.html#building-against-numpy).

Comment (christianbrodbeck) [reply recipes/eelbrain/meta.yaml]: This says the default for Unix would currently be `1.9.3`, so I think it still needs to be pinned to `1.11`, right?

Comment (chrisburr) [reply recipes/eelbrain/meta.yaml]: Yup, you're entirely correct. Sorry for the mix up.

**Comment by christianbrodbeck** [comment]
Thanks @chrisburr !

**Pull request merged**